test/integrationtests/test_constants.go

```diff
@@ -0,0 +1,18 @@
+package integrationtests
+
+// Scopes.
+const (
+	MsIDlabDefaultScope = "https://msidlab.com/.default"
```

Comment:
There doesn't look to be a good reason to have a file with just constants in it.  I'd simply create a "integrations.go" file and put it in there along with all your tests from user_password_test.go until the file gets somewhere between 500 and 1000 lines before I'd break it up.

Also, may of these constants are exported(public), they should probably be private.